versions/releases.py

```diff
@@ -226,17 +267,15 @@ def store_release_notes_for_version(version_pk):
         )
         raise Version.DoesNotExist
 
-    # Get the release notes content
-    content = get_release_notes_for_version(version_pk)
-    stripped_content = process_release_notes(content)
+    content, processed_content, content_type = get_release_notes_for_version(version_pk)
 
     # Save the result to the rendered content model with the version cache key
     rendered_content, _ = RenderedContent.objects.update_or_create(
```

Comment:
There doesn't appear to be any handling of the case where we fail to get the release notes - maybe I'm missing it, though?

Reply:
There isn't, could be changed but this is identical to the previous behavior of `get_release_notes_for_version`.
This function is called in a celery task so if this function fails, just that task will fail.

Reply:
OK that's fine - we'll leave it for now.